Samples/Samples/App.xaml.cs

```diff
@@ -48,19 +56,33 @@ protected override async void OnStart()
                 typeof(Distribute));
             }
 
-            await AppActions.SetAsync(
-                new AppAction("app_info", "App Info", icon: "app_info_action_icon"),
-                new AppAction("battery_info", "Battery Info"));
+            try
+            {
+                await AppActions.SetAsync(
+                    new AppAction("app_info", "App Info", icon: "app_info_action_icon"),
+                    new AppAction("battery_info", "Battery Info"));
+            }
+            catch (FeatureNotSupportedException ex)
+            {
+                Debug.WriteLine($"{nameof(AppActions)} Exception: {ex}");
+            }
         }
 
         void AppActions_OnAppAction(object sender, AppActionEventArgs e)
         {
             // Don't handle events fired for old application instances
             // and cleanup the old instance's event handler
-            if (Application.Current != this && Application.Current is App app)
+            try
```

Comment:
This does not need to be wrapped in a try/catch as the actual registration check will catch it.

Reply:
I must be tired.